backend/server/models/studentinstance.js

```diff
@@ -21,6 +21,11 @@ module.exports = (sequelize, DataTypes) => {
         allowNull: false,
         defaultValue: false
       },
+      valid: {
```

Comment:
I think this column should be named `registrationValid` (or something like that) as `valid` is a bit vague term.

Reply:
Yes i was actually thinking the same, I will change it to be descriptive. Thanks!